utils/common_validation.go

```diff
@@ -0,0 +1,53 @@
+package utils
```

Comment:
I think utils should be injected as interface to service struct for mocking and testing sakes.

Reply:
Can we accept this as is until IEP-634 is ready to go as no idea yet how to implement that